mm2src/adex_cli/src/cli.rs

```diff
@@ -0,0 +1,113 @@
+use clap::{App, Arg, SubCommand};
+use log::error;
+use std::env;
+
+use crate::scenarios::{get_status, init, start_process, stop_process};
+
+enum Command {
+    Init {
+        mm_coins_path: String,
+        mm_conf_path: String,
+    },
+    Start {
+        mm_conf_path: Option<String>,
+        mm_coins_path: Option<String>,
+        mm_log: Option<String>,
+    },
+    Stop,
+    Status,
+}
+
+pub fn process_cli() {
+    let mut app = App::new(env!("CARGO_PKG_NAME"))
+        .version(env!("CARGO_PKG_VERSION"))
+        .author(env!("CARGO_PKG_AUTHORS"))
+        .about(env!("CARGO_PKG_DESCRIPTION"))
+        .subcommand(
+            SubCommand::with_name("init")
+                .about("Initialize predefined mm2 coin set and configuration")
+                .arg(
+                    Arg::with_name("mm-coins-path")
+                        .long("mm-coins-path")
+                        .value_name("FILE")
+                        .help("coin set file path")
+                        .default_value("coins"),
+                )
+                .arg(
+                    Arg::with_name("mm-conf-path")
+                        .long("mm-conf-path")
+                        .value_name("FILE")
+                        .help("mm2 configuration file path")
+                        .default_value("MM2.json"),
+                ),
+        )
+        .subcommand(
+            SubCommand::with_name("start")
+                .about("Start mm2 service")
+                .arg(
+                    Arg::with_name("mm-conf-path")
+                        .long("mm-conf-path")
+                        .value_name("FILE")
+                        .help("mm2 configuration file path"),
+                )
+                .arg(
+                    Arg::with_name("mm-coins-path")
+                        .long("mm-coins-path")
+                        .value_name("FILE")
+                        .help("coin set file path"),
+                )
+                .arg(
+                    Arg::with_name("mm-log")
+                        .long("mm-log")
+                        .value_name("FILE")
+                        .help("log file path"),
+                ),
+        )
+        .subcommand(SubCommand::with_name("stop").about("Stop mm2 service"))
```

Comment:
It's possible using different `rpc_port` and `db_dir`, but it's not supposed to start multiple using `adex`? 
I would not confuse anybody, trying to be simple and stupid. The regular use is not aware about "instance", they aware about "service" it brings more common sense. 

> E.g. on macos start multiple instances would be refused by launchctl.  
> If we had multiple instances this command would stop them all.

Reply:
Nop, I do not think this about being "simple" or "common sense" or "stupid".

In some cases, running multiple instances of a CLI with different configurations may be necessary or useful. For example, if you are working with multiple nodes that use the same CLI, you may want to run separate instances of the CLI with different configurations for each node.

Reply:
We can have problems having multiple instances
https://github.com/KomodoPlatform/atomicDEX-API/pull/1694
https://github.com/KomodoPlatform/atomicDEX-API/pull/1704#discussion_r1138546265

Reply:
These are both good enough and worth to be there )
I don't mind if it would be "instance"

Reply:
> We can have problems having multiple instances #1694 [#1704 (comment)](https://github.com/KomodoPlatform/atomicDEX-API/pull/1704#discussion_r1138546265)

It could be useful for test purposes

Reply:
uhmmm what I was trying to suggest is that, if we can have the `adex-cli` running in multiple instances then we should use the word "instance" instead of "service"